### ISSUE TITLE
stl files of parts

### ISSUE THREAD

**Issue by joshpunb** [jochenalt/Walter#1]
Would it be possible to include the stl files for all of the parts? 

Also, what version of TurboCAD can open the tcw files? I try to open them with TurboCAD 2016 deluxe and get the following warning:
"The drawing may contain object(s) that are not supported
by the current version of TurboCAD.
Such object(s) can be load but will not be shown.
Do you want to store them in the drawing?"

**Comment by oudent** [comment]
TurboCAD 2016 Deluxe clearly said I needed TurboCAD Professional to open the parts. There is a free trial available. However, have you tried filtering through the DWG files? But ya, stl's would be nice, if anything just to know exactly which parts are needed, as the DWG files have a lot of extra weirdness in them, at least when opened in Fusion360.

**Comment by jochenalt** [comment]
Yes,it is TurboCad 22 pro. I would say it is best to download the trial and try out which exchange format works best. STL is not apprpriate for exchange, since all geometric stuff is gone. The dwg files are for autocad/ inventor, this works for most CADs.

**Comment by joshpunb** [comment]
Thanks. The files open properly in TurboCAD Pro Platinum 2017.
